.github/workflows/submit.yml

```diff
@@ -13,8 +13,6 @@ jobs:
       collection: '2025'
       kind: original
       path: papers/*
-      label: publish
```

Comment:
`label` only works for actions on `pull_request`; for `push` there are no labels to check.